public/js/src/module/gui.js

```diff
@@ -184,56 +180,18 @@ function swapTheme(formParts) {
     });
 }
 
-feedbackBar = {
-    /**
-     * Shows an unobtrusive feedback bar to the user.
-     *
-     * @param { string } message - message to show
-     * @param {number=} duration - duration in seconds for the message to show
-     */
-    show(message, duration) {
-        let $msg;
-        const $fbBar = $('#feedback-bar');
-
-        duration = duration ? duration * 1000 : 10 * 1000;
```

Comment:
The actual change could be achieved by removing this line and wrapping the timeout below in `if (!duration)`.

However, I was a bit embarrassed by this old code and thought I should do a good deed and refactor it as well.